root/url/utility/isGreyedOut.js

```diff
@@ -7,10 +7,21 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-const greyRegExp = new RegExp('^https?://(?:www\.)?decoda\.com/');
+const MALWARE_URLS = [
+  'decoda\\.com',
+  'starzik\\.com',
+].map(host => new RegExp('^https?://([^/]+\\.)?' + host + '/?(.*)$', 'i'));
```

Comment:
For second commit https://github.com/metabrainz/musicbrainz-server/pull/1894/commits/6d1ebe2a297c31ccf967d948fe2f4e55b1eb7fe6:

```suggestion
].map(host => new RegExp('^https?://([^/]+\\.)?' + host + '(/.*)?$', 'i'));
```